src/views/Languages/index.jsx

```diff
@@ -1,69 +1,266 @@
+import projects from '../../data/loader';
 
 const getIgnoreCase = (object, keyToFind) => {
   const key = Object.keys(object).find(key => key.toLowerCase() === keyToFind);
 
   return key && object[key];
 };
 
+const repos = mergeAll(
+  Object.values(projects)
+    .filter(project => project.repositories)
+    .map(project => project.repositories)
+    .reduce((prev, curr) => [...prev, ...curr], [])
+);
+const tagReposMapping = repositories =>
+  Object.keys(repositories).reduce((previousMappings, repoName) => {
+    const tags = repositories[repoName];
+    const labels = typeof tags === 'string' ? [tags] : tags;
+    const mappings = labels.reduce(
+      (labels, label) => ({
+        ...labels,
+        [label]: [...(previousMappings[label] || []), repoName],
+      }),
+      {}
+    );
+
+    return {
+      ...previousMappings,
+      ...mappings,
+    };
+  }, {});
+
 @withApollo
 @hot(module)
-@graphql(bugsQuery, {
-  skip: ({
-    match: {
-      params: { language },
-    },
-  }) => !getIgnoreCase(BUGZILLA_LANGUAGES, language),
-  name: 'bugzilla',
-  options: ({
-    match: {
-      params: { language },
-    },
-  }) => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      goodFirst: {
-        ...BUGZILLA_SEARCH_OPTIONS,
-        keywords: [GOOD_FIRST_BUG],
-        whiteboards: `lang=${getIgnoreCase(BUGZILLA_LANGUAGES, language)}`,
+@compose(
+  graphql(issuesQuery, {
+    name: 'github',
+    options: ({
+      match: {
+        params: { language },
+      },
+    }) => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: [
+          `language:${language}`,
+          ...Object.keys(repos).map(repo => `repo:${repo}`),
+        ].join(' '),
+        order: GITHUB_LANGUAGE_QUERY_ORDER,
+        type: 'REPOSITORY',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
+  }),
+  graphql(bugsQuery, {
+    skip: ({
+      match: {
+        params: { language },
       },
-      mentored: {
-        ...BUGZILLA_SEARCH_OPTIONS,
-        ...MENTORED_BUG,
-        whiteboards: `lang=${getIgnoreCase(BUGZILLA_LANGUAGES, language)}`,
+    }) => !getIgnoreCase(BUGZILLA_LANGUAGES, language),
+    name: 'bugzilla',
+    options: ({
+      match: {
+        params: { language },
       },
-      paging: {
-        ...BUGZILLA_PAGING_OPTIONS,
+    }) => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        goodFirst: {
+          ...BUGZILLA_SEARCH_OPTIONS,
+          keywords: [GOOD_FIRST_BUG],
+          whiteboards: `lang=${getIgnoreCase(BUGZILLA_LANGUAGES, language)}`,
+        },
+        mentored: {
+          ...BUGZILLA_SEARCH_OPTIONS,
+          ...MENTORED_BUG,
+          whiteboards: `lang=${getIgnoreCase(BUGZILLA_LANGUAGES, language)}`,
+        },
+        paging: {
+          ...BUGZILLA_PAGING_OPTIONS,
+        },
       },
+      context: {
+        client: 'bugzilla',
+      },
+    }),
+  })
+)
+@withStyles(theme => ({
+  root: {
```

Comment:
There seems to be a lot of unused styles in this file. Probably from the rebase.